client/rest-high-level/src/main/java/org/elasticsearch/client/ml/dataframe/OutlierDetection.java

```diff
@@ -47,6 +47,8 @@ public static Builder builder() {
     public static final ParseField NAME = new ParseField("outlier_detection");
     static final ParseField N_NEIGHBORS = new ParseField("n_neighbors");
     static final ParseField METHOD = new ParseField("method");
+    public static final ParseField MINIMUM_SCORE_TO_WRITE_FEATURE_INFLUENCE =
```

Comment:
AFAIU this does not have to be "public".

Reply:
Ah, that reminded me I should have included this field in `ReservedFieldNames`! Thanks!

Reply:
Still, the "client" version of this field can be package-private. Only "backend" version has to be public.

Reply:
I think usually we make those parse fields public as there is no conceptual reason to hide them. For client, for example, if for some reason a user that writes some integration wanted to, say, map some custom config setting to this, it would be nice to let them refer via the constant instead of forcing them to use a string literal. On the other hand, I do not see any downside exposing those.

Reply:
SGTM